pkg/rest/apisurface.go

```diff
@@ -69,7 +69,7 @@ func (s *APISurface) ProvisionHandler(w http.ResponseWriter, r *http.Request) {
 
 	request, err := unpackProvisionRequest(r)
 	if err != nil {
-		writeError(w, err, http.StatusInternalServerError)
+		writeError(w, err, http.StatusBadRequest)
```

Comment:
Should we also do the same below when the provision returns the error?

Reply:
Good question - the spec isn't super clear with a suggestion of another error code to use.  If you return 500, it should trigger the orphan mitigation behavior. How about this - let's think about what the default error code should be (since you can specify your own - the codes in provision, for example, are just defaults) and change it in a follow up.  For now, at least we know that the current behavior is tested.

WDYT?

Reply:
Yep, that sounds good.